product_packaging_dimension/models/product_packaging.py

```diff
@@ -0,0 +1,27 @@
+# Copyright 2019 Camptocamp SA
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl)
+from odoo import api, fields, models
+
+
+class ProductPackaging(models.Model):
+    _inherit = "product.packaging"
+
+    weight = fields.Float('Weight (kg)')
```

Comment:
:warning: We might have to drop commits 4d877ab and 6bfbea7 because `max_weight` is defined in `delivery`: https://github.com/odoo/odoo/blob/12.0/addons/delivery/models/product_packaging.py#L13

Reply:
AFAIK having 2 modules defining the same field is not really a problem.